scripts/get_gnomad_lof_variants.py

```diff
@@ -178,7 +240,13 @@ def get_gnomad_lof_variants(gnomad_version, gene_ids, include_low_confidence=Fal
     ds = ds.select(
         reference_genome="GRCh37" if gnomad_version == 2 else "GRCh38",
         variant_id=variant_id(ds.locus, ds.alleles),
-        liftover_variant_id=hl.missing(hl.tstr),
+        liftover_variant_id=hl.if_else(
```

Comment:
It's hard to follow these multiple `is_missing` and `if_else` calls here, introducing a few intermediate variables might be the thing to clarify this bit.